code/modules/mob/living/carbon/xenomorph/castes/hivemind/castedatum_hivemind.dm

```diff
@@ -75,6 +76,7 @@
 		/datum/action/xeno_action/plant_weeds/slow,
 		/datum/action/xeno_action/psychic_whisper,
 		/datum/action/xeno_action/activable/psychic_cure,
+		/datum/action/xeno_action/activable/rally_hive,
```

Comment:
These all need to be the hivemind version, otherwise it will revert to the parent version upon upgrade.

unless this is a deliberate choice.